tests/core/test_flow.py

```diff
@@ -1836,6 +1836,56 @@ def test_serialize_fails_with_no_storage(self):
             s_build = f.serialize(build=True)
 
 
+class TestSerializedHash:
```

Comment:
We could also test across python runs with `python -c "from prefect import Flow; print(Flow('test').serialized_hash())"` (which I did manually to make sure) but I'm not sure it's worth the hassle of making sure we're spawning a process with the right python executable? / does this hash really need a ton of coverage for `flow.serialize()` features?

Reply:
I don't think that's necessary - ultimately the worst case for the hash changing is a new registered Flow version, so just ensuring that it's reasonably consistent is enough IMO.